lib/ui/window/window.cc

```diff
@@ -142,6 +142,30 @@ void _RespondToPlatformMessage(Dart_NativeArguments args) {
   tonic::DartCallStatic(&RespondToPlatformMessage, args);
 }
 
+void GetPersistentIsolateData(Dart_NativeArguments args) {
+  auto persistent_isolate_data =
+      UIDartState::Current()->window()->client()->GetPersistentIsolateData();
+
+  if (!persistent_isolate_data) {
+    Dart_SetReturnValue(args, Dart_Null());
+    return;
+  }
+
+  auto data_handle = Dart_NewTypedData(Dart_TypedData_kByteData,
```

Comment:
Reuse `ToByteData`here.  `ToByteData` can use `void*` and `length` arguments, and you can add wrappers that take an `fml::Mapping` or `vector`

Tonic's `DartByteData` class might also be a good place for this helper function.

Reply:
Done.